v2/azure-arm.yaml

```diff
@@ -984,6 +984,34 @@ objectModelConfiguration:
           $armReference: true
         PrivateDnsZoneArmResourceId:
           $armReference: true
+  devices:
+    2021-07-02:
+      IotHub:
```

Comment:
Should we also support EventHubConsumerGroups? That seems to be the other main IOTHub resource I can see in their Swagger.